website/views.py

```diff
@@ -4332,3 +4334,17 @@ def deletions(request):
         "monitor.html",
         {"form": form, "deletions": Monitor.objects.filter(user=request.user)},
     )
+
+
+def generate_bid_image(request, bid_amount):
+    image = Image.new("RGB", (100, 50), color="white")
+    draw = ImageDraw.Draw(image)
+
+    font = ImageFont.load_default()
+    draw.text((10, 10), f"Bid: ${bid_amount}", fill="black", font=font)
+
+    byte_io = io.BytesIO()
+    image.save(byte_io, format="PNG")
+    byte_io.seek(0)
+
+    return HttpResponse(byte_io, content_type="image/png")
```

Comment:
We can use fileresponse 
But can you tell me why is it better , thank you

Reply:
I recommended it as an alternative which would allow us simplify the controller logic by using `FileResponse` abstraction (it accepts any file-like object with binary content). It fitted the use-case and the main idea was to get rid of the `io` import if possible.

Reply:
Okay will change it thank you !